kroxylicious/src/main/java/io/kroxylicious/proxy/config/admin/AdminHttpConfiguration.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.proxy.config.admin;
+
+public class AdminHttpConfiguration {
+    private final String host;
+    private final Integer port;
```

Comment:
Can we give give these default values?

Reply:
changed to have a default of `0.0.0.0` and `9193`

Reply:
Thanks!